Symfony/CS/Fixer.php

```diff
@@ -82,13 +89,17 @@ public function __construct()
     public function registerBuiltInFixers()
     {
         foreach (Finder::create()->files()->in(__DIR__.'/Fixer') as $file) {
+            /* @var FinderSplFileInfo $file */
```

Comment:
this needs to be ABOVE the foreach, and MUST be `/**` style, otherwise it's not even phpdoc